core/bentley/src/Expect.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * Expects that a value is defined (not undefined).
+ * @note Almost always, places that use this function should in the future be cleaned up to properly
+ * handle undefined values instead of throwing an error.
+ * @param value The value to check.
+ * @param message Optional error message to be used in thrown Error.
+ * @returns The original value if it is defined.
+ * @throws Error if the value is undefined.
+ * @beta
```

Comment:
```suggestion
 * @internal
```

We don't want anybody outside of itwinjs-core using this, right? We want to use it temporarily and then remove it ASAP after cleaning up all the usages.